common/test/java/com/couchbase/lite/CollectionTest.kt

```diff
@@ -66,6 +66,20 @@ class CollectionTest : BaseCollectionTest() {
         }
     }
 
+    // Test get doc from collection that is deleted in a different database instance
```

Comment:
Again... It would be so much more helpful if the comment gave the expected result: "verify that getting a document that has been deleted from a different database causes a CBL exception"